README.adoc

```diff
@@ -1808,6 +1808,36 @@ date.all_quarter
 date.all_year
 ----
 
+=== Comparison
+
+Avoid comparing time with `>` and `<` to avoid confusion.
+
+[source,ruby]
+----
+# bad
+created_at < 5.minutes.ago
+shutdown_at > 1.minute.from_now
+
+# good
+created_at.before?(5.minutes.ago)
+shutdown_at.after?(1.minute.from_now)
+----
+
+Use `past?` and `future?` when comparing with the current time.
```

Comment:
👍 totally agree with this part.

Reply:
I'm fine with doing the prime time for `past?`/`future?`, and just briefly mentioning `before?`/`after?` below the main section with no guidance to use. WDYT?

Reply:
Agreed.